package.json

```diff
@@ -17,18 +17,18 @@
     "axios": "^0.21.1",
     "bootstrap": "4.5.3",
     "node-sass": "4.14.1",
-    "react": "^16.3.1",
```

Comment:
Put the v16 back and remove the v17. Or copy and paste the package.json file from the team's develop branch.